### DISCUSSION TITLE
Shaders and Ressource pack downloader ?

### DISCUSSION THREAD

**Discussion by SpidFightFR** [PrismLauncher/PrismLauncher#718] [Ideas]
Hey guys, the mod downloader and modpack downloader both work amazingly, but it would be perfect if Prism had a similar system for ressource packs and shaders, don't you think ?

**Comment by DioEgizio** [comment]
It's planned, there's already some preliminary work for it and there's even someone working on it

**Reply by SpidFightFR** [reply]
noice!

**Reply by roke-julian-lockhart** [reply]
> https://github.com/orgs/PrismLauncher/discussions/718#discussioncomment-4624812

@DioEgizio, is it not already implemented in https://github.com/PrismLauncher/PrismLauncher/releases/tag/8.3?

> ![Screenshot_20240621_001832](https://github.com/PrismLauncher/PrismLauncher/assets/42837531/1748b96b-5e1f-4f5f-80f1-853ea77df075)

**Reply by Trial97** [reply]
Yes they are implemented
